taskboot/push.py

```diff
@@ -130,6 +130,9 @@ def heroku_release(target, args):
     # Trigger a release on Heroku
     logger.info("Deploying update for dyno types: %r", list(sorted(x["type"] for x in updates_payload)))
 
+    updates_payload = {"updates": updates_payload}
+    logger.info("Using payload: %r", updates_payload)
```

Comment:
Either remove or set it as debug, i don't like having the full output with Application data exposed by default.

Reply:
I have moved the two log message to the debug level. The task-boot CLI don't have yet a way to set the log level, should we introduce a `-v` or `--debug` CLI flag? What do you think?

Reply:
In another PR, i filed: https://github.com/mozilla/task-boot/issues/50